packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -5232,6 +5236,7 @@ export class Bn254Add extends Op {
 		return this.computeCost();
 	}
 }
+//TODO:add description
```

Comment:
You can add them now.

Reply:
this is for another opcode. I just added it there so we do not forget about it. This opcode is not ready yet